index.html

```diff
@@ -7,9 +7,15 @@
   <body>
     <form class="form-wrapper">
       <div class="form-section">
+        <img
+          class="mobile-display"
+          src="./assets/harmony-music-academy-logo.svg"
+          alt="Harmony Music Academy logo"
+        />
         <div class="form-field">
           <label for="email" class="form-label">Email*</label>
           <input
+            id="email"
```

Comment:
Good - don't forget your select box and your `hoursPerWeek` input field need unique ids adding as well.